src/main/java/suppliers/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package suppliers;
+
+import core.basesyntax.model.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int colorIndex = new Random().nextInt(Color.values().length - 1);
```

Comment:
Let's create an object of class Random only once, and store it as a class field
```suggestion
        int colorIndex = random.nextInt(Color.values().length - 1);
```